src/sdk/sql_router.h

```diff
@@ -71,6 +71,8 @@ class SQLRouter {
     virtual ~SQLRouter() {}
 
     virtual bool ShowDB(std::vector<std::string>* dbs, hybridse::sdk::Status* status) = 0;
+    
+    virtual std::vector<std::string> GetAllTables(hybridse::sdk::Status* status) = 0;
```

Comment:
use GetTableNames

Reply:
We already have `GetTableNames` function that return tables of specific dbs.
`GetAllTables` is to help create new interface for supporting more Python SDK.

Reply:
You can get all db by `ShowDB` at first

Reply:
yes :) that's how we realize it in `GetAllTables` in `sql_cluster_router.cc`